ccp/apis/v1/ccp_server/env_config.py

```diff
@@ -6,4 +6,5 @@
 JENKINS_URL = os.environ.get("JENKINS_URL")
 INDEX_GIT_URL = os.environ.get("INDEX_GIT_URL")
 INDEX_GIT_BRANCH = os.environ.get("INDEX_GIT_BRANCH")
+INDEX_CLONE_LOCATION = "/var/index/repo"
```

Comment:
Why are we cloning it in `/var`? It's a directory to which system normally writes. Our service is running on top of it so IMO we should be using `/tmp` for cloning repo.

Reply:
ACK, my idea was to put it any where. yeah makes sense. changing it.